.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,20 @@
+name: Lighthouse check
+on: [ push ]
+jobs:
+  lhci:
+    name: Lighthouse
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install & Build
+        run: |
+          yarn install
+          yarn build
+      - name: Lighthouse Check
+        run: |
+          npm install -g @lhci/cli@0.8.x
+          lhci autorun
```

Comment:
It's in a completely different file with its own build process. The build process may be costly on other projects because it will simply take more time to execute it.

And I want to have another task in the master action 😅 

We could automate it by putting this `lhci` task into the `master.yml` file after the build and get the generated artifact with the website from the `build` task.